src/lib/yang/snabb-softwire-v1.yang

```diff
@@ -19,6 +19,14 @@ module snabb-softwire-v1 {
     description
      "Configuration for Snabb lwaftr.";
 
+     leaf name {
```

Comment:
Consider the case of wanting to run a number of Snabb applications on the command line with the same configuration (binding table etc) but different NICs.  Probably you want different names there too.  As PCI addresses for NICs are passed on the command line, I think users will also want to pass names on the command line.  Dunno.  Also if we include a name here in the YANG schema then we'd want "snabb config set /name foo" to work, and I think that maybe doesn't make sense.  WDYT?  We could certainly have name as state data of course, though that runs into other problems.

In short I suspect that passing the name on the command line is the most useful way to work.  WDYT?